.vscode/tasks.json

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
oops! I don't think you meant to add this vscode file, although it's kind of fun to look at (I don't use vscode but I can see how it works a bit now!)

Reply:
ahah funny indeed, didn't notice.
I didn't write it, it's the application that creates it for its own wizardries